src/query/interpreter.cpp

```diff
@@ -1212,6 +1226,10 @@ PreparedQuery Interpreter::PrepareTransactionQuery(std::string_view query_upper)
       if (!in_explicit_transaction_) {
         throw ExplicitTransactionUsageException("No current transaction to rollback.");
       }
+
+      utils::OnScopeExit rollback_metric_update(
+          []() { memgraph::metrics::IncrementCounter(memgraph::metrics::RollbackedTransactions); });
+
```

Comment:
Abort isn't noexcept but I think but if you put it before Abort() call that's also fine.